### PR TITLE
Feature/change namespace

### PR DESCRIPTION
**Pull request by thomasnordahl-dk**
Composer should allow 2 packages to share a namespace, but it's not optimal or predictable, so I propose a new namespace for this package.

Since we are breaking backwards compatibility, I thought I might throw out PHP 7.x support. I'll submit another PR with updated typehints, if we agree to skip 7.x support.

### CODE REVIEW

File: UPGRADING.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+From version 2.0.0, `kodus/sql-split` requires at least PHP version 8.0
+
+### Namespace changed
+From version 2.0.0, the namespace of the `Splitter` and `Tokenizer` has changed from `Kodus\SQL` to `Kodus\SQLSplit`.
+
+```diff
+<?php
+
+-use Kodus\SQLSplit\Splitter;
++use Kodus\SQLSPlit\Splitter;
```
Comment (boan-jfm) [inline UPGRADING.md]:
```suggestion
+use Kodus\SQLSplit\Splitter;
```

File: UPGRADING.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+-use Kodus\SQLSplit\Tokenizer;
++use Kodus\SQLSPlit\Tokenizer;
```
Comment (boan-jfm) [inline UPGRADING.md]:
```suggestion
+use Kodus\SQLSplit\Tokenizer;
```

**Review by boan-jfm — APPROVED** [review]

File: UPGRADING.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+UPGRADING
+=========
+## 1.x &rarr; 2.x
+
+### PHP 8.0 required
```
Comment (JyskFynskeMedierJoej) [inline UPGRADING.md:5]: Should we maybe confirm with OPS that we do not have a server that cannot meet this requirement

**Review by JyskFynskeMedierJoej — APPROVED** [review]

File: UPGRADING.md (hunk 3 above)
Comment (thomasnordahl-dk) [reply UPGRADING.md:5]: I'm thinking that if we have servers with projects still using older PHP versions, the projects should declare the PHP requirements in composer.json correctly, so dependencies are not upgraded to this version prematurely.

Comment (JyskFynskeMedierJoej) [reply UPGRADING.md:5]: Ok

**Review by JyskFynskeMedierJoej — APPROVED** [review]

**Review by JyskFynskeMedierJoej — APPROVED** [review]

**Pull request merged**